features/adb-openshift.feature

```diff
@@ -57,6 +57,14 @@ Feature: Command output from various OpenShift related commands in ADB
     Then the exit status should be 0
     And the binary "oc" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli openshift --cli-version 1.3.0`
+    Then the exit status should be 0
+    And the binary "oc" of service "openshift" should be installed with version "1.3.0"
```

Comment:
👍 

I created issue #415 to follow up on what's going on with CI and why we did not get test failures when some of the tests were clearly wrong. We need to get to the bottom of this, otherwise we cannot trust CI results, in particular since these tests are not run per default on the dev machine